Extension fails to load when being run in a Dev Container

### Describe the bug
The extension fails to load when being run in a Dev Container, as `bracket-pair-colorizer-2` is looking for a `node_modules.asar` folder which does not exist in this environment, crashing the extension on load.

### To Reproduce
A simple reproduction repo is available [here](https://github.com/tom-fletcher/vscode-better-folding-repro), with instructions in the README.

### Expected behaviour
The extension loads.

### Root cause
The cause of the bug can be seen when viewing the output for the `Remote Extension Host` which shows the extension failed to load with the following error:

```
2023-01-31 23:11:06.095 [error] Activating extension MohammadBaqer.better-folding failed due to an error:
2023-01-31 23:11:06.096 [error] Error: Cannot find module '/vscode/vscode-server/bin/linux-x64/97dec172d3256f8ca4bfb2143f3f76b503ca0534/node_modules.asar/vscode-textmate'
```
When checking the file system, the directory `.../node_modules.asar/...` does not exist. However, in the same path the directory `.../node_modules/...` does exist.

The issue lies with this function in `brackets-pair-colorizer-2 src`:
https://github.com/mtbaqer/vscode-better-folding/blob/116ddd2c0f031720da1734b48b95a5c9f135a72f/src/bracket-pair-colorizer-2%20src/textMateLoader.ts#L118-L120

In a Dev Container, this folder does not have the `.asar` suffix. I did a proof-of-concept check, changing this from `node_modules.asar` to `node_modules` then recompiling and re-installing the extension. This worked and the extension loaded as expected in the Dev Container *(note that the fixes only showed on new files, and I had to re-create `test.js` before folding worked correctly)*.

### Suggested fix

My suggested fix would be to modify the function so it checks whether the `.asar` folder is present:
```diff
 private getNodeModulePath(moduleName: string) { 
-  return path.join(vscode.env.appRoot, "node_modules.asar", moduleName); 
+  return fs.existsSync(path.join(vscode.env.appRoot, "node_modules.asar"))
+    ? path.join(vscode.env.appRoot, "node_modules.asar", moduleName)
+    : path.join(vscode.env.appRoot, "node_modules", moduleName);
 } 
```

Also happy to submit this fix via PR if this helps.

--- a/src/bracket-pair-colorizer-2 src/textMateLoader.ts
+++ b/src/bracket-pair-colorizer-2 src/textMateLoader.ts
@@ -116,7 +116,9 @@ export class TextMateLoader {
   }
 
   private getNodeModulePath(moduleName: string) {
-    return path.join(vscode.env.appRoot, "node_modules.asar", moduleName);
+    return fs.existsSync(path.join(vscode.env.appRoot, "node_modules.asar"))
+      ? path.join(vscode.env.appRoot, "node_modules.asar", moduleName)
+      : path.join(vscode.env.appRoot, "node_modules", moduleName);
   }
 
   private getNodeModule(moduleName: string) {

